client/src/components/InstanceSummaryView.js

```diff
@@ -102,7 +102,7 @@ const InstanceSummaryView = () => {
       </Box>
       <Box sx={{ display: 'flex', flexWrap: 'wrap', justifyContent: 'space-between', alignItems: 'center', pb: 6 }}>
         <Button variant='outlined' onClick={() => onGoBack()} disabled={messageInfo?.severity === 'info'} >Go back</Button>
-        <Button variant='contained' onClick={() => onCreateInstance()} disabled={messageInfo?.severity === 'info'}>Create instance</Button>
+        <Button id='ag_create_instance_btn' variant='contained' onClick={() => onCreateInstance()} disabled={messageInfo?.severity === 'info'}>Create instance</Button>
```

Comment:
Should separate these onto individual rows to make the line length a bit shorter